tcms/templates/include/tc_executions.html

```diff
@@ -28,23 +28,24 @@ <h2 class="card-pf-title">
                                     <a href="{% url 'test_plan_url_short' execution.run.plan.pk %}">TP-{{ execution.run.plan.pk }}: {{ execution.run.plan.name }}</a>
                                 </div>
                                 <div class="list-group-item-text">
-                                    <div class="completion-rate-container">{% trans 'Completion rate' %}: <span class="completion-rate"></span> 
+                                    <div class="completion-rate-container">{% trans 'Completion rate' %}: <span class="completion-rate"></div>
```

Comment:
This is wrong. You are missing the closing `</span>` tag which totally destroys how this is rendered. As a result I can't verify the calculation bug either.  This is how it looks now:

![Screenshot_2021-09-08 Kiwi TCMS - TC-1 Login with valid credentials works](https://user-images.githubusercontent.com/1002300/132567462-c839d2e6-1cee-4293-8dea-a638821b040a.png)

Reply:
it should work OK now.  however, I don't think we should make this look like Execution trends. In Execution trends we have 3 types of statuses, that depend on one another, e.g. the more FAILs we have, the less SUCCESS we have.

Here, we have 2 metrics that are independent, we could have 100% completion rate and 100% failure rate, but we could also have 100% completion rate and 0% failure rate.